CONTRIBUTING.md

```diff
@@ -8,38 +8,60 @@ Please note that all activity on the [`pikapkg/snowpack` repository](https://git
 
 Our [issue tracker](https://github.com/pikapkg/snowpack/issues) is always organized with a selection of high-priority bugs, feature requests, and ["help wanted!"](https://github.com/pikapkg/snowpack/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)/["good first issue"](https://github.com/pikapkg/snowpack/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) items. For general package troubleshooting and discussions, check out our [Package Community](https://www.pika.dev/npm/snowpack/discuss) discussion board.
 
+## Requirements
+
+Snowpack uses [yarn workspaces](https://classic.yarnpkg.com/) to manage multiple packages contained in this repository. To contribute, [make sure that you have Yarn installed on your machine](https://classic.yarnpkg.com/en/docs/install).
+
+## Initial setup
+
 ```bash
-# Local Setup
-git clone ${REPO}
+git clone https://github.com/pikapkg/snowpack.git
+cd snowpack
 yarn
 ```
 
+## Build after changes
+
+Note: you will see warnings about `__dirname` and `require()` not being "a valid ESM global" when running `yarn build`. You can ignore these warnings.
+
 ```bash
-# Build after changes
 yarn build
 yarn --force
 ```
 
-```bash
-# Check formatting after changed
-yarn format
-```
+## Run tests
 
 ```bash
-# Run your tests
 yarn build
 yarn --force
 yarn test
 ```
 
+You can filter the tests that are being run using Jest's [`--testNamePattern`](https://jestjs.io/docs/en/cli#--testnamepatternregex) CLI option. You can ignore the `123 snapshots obsolete` messages.
+
+```bash
+yarn test --testNamePattern treeshake
```

Comment:
nit: I'd love a mention somewhere here that `-t` is also supported. Something like:

> You can filter the tests that are being run using Jest's `--testNamePattern` (alias: `-t`) CLI option.